includes/class-debug.php

```diff
@@ -55,6 +54,24 @@ public static function log_inbox( $data, $user_id, $type ) {
 		}
 	}
 
+	/**
+	 * Log the sent to follower action.
+	 *
+	 * @param array  $result The result of the remote post request.
+	 * @param string $inbox  The inbox URL.
+	 * @param int    $actor_id The ID of the actor.
+	 * @param array  $activity The activity.
+	 */
+	public static function log_sent_to_follower( $result, $inbox, $actor_id, $activity ) {
+		if ( \is_wp_error( $result ) ) {
```

Comment:
`$result` will be a `WP_REST_Response` object, not a `WP_Error`. It does have[ a `as_error()` method](https://github.com/WordPress/wordpress-develop/blob/6.7/src/wp-includes/rest-api/class-wp-rest-response.php#L225-L246) though, that helps with error handling.
So you could convert it first and then check it with `is_wp_error()`

Reply:
this is the custom function and it returns an error 🫣

https://github.com/Automattic/wordpress-activitypub/blob/8c9f6a842e02d89e573073bedd2bf3aaae9f9f20/includes/class-http.php#L68

Reply:
Oh! We could consider updating that to return the actual error that's being returned